src/OpenTelemetry.Instrumentation.Process/README.md

```diff
@@ -46,7 +46,23 @@ using var meterProvider = Sdk.CreateMeterProviderBuilder()
 ```
 
 Refer to [Program.cs](../../examples/process-instrumentation/Program.cs) for a
-complete demo.
+complete demo. This examples sets up the OpenTelemetry Prometheus exporter,
+which requires adding the package [`OpenTelemetry.Exporter.Prometheus`][2] to
+the application.
+
+Additionally, this [document][3] shows how to use Prometheus and Grafana to
+build a dashboard for your application. [This][4] is the Grafana dashboard
+template which has all the metrics currently supported by this package; plus
+an additional aggregated metric `CPU utilization` calculated with the raw
+metrics, `CPU time` and `CPU count`.
+
+Please follow the instructions in this [document][5] to import a Grafana
+dashboard by uploading the JSON template file.
+
+[2]: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.HttpListener/README.md
+[3]: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/getting-started-prometheus-grafana/README.md
+[4]: ../../examples/process-instrumentation/process-instrumentation-grafana-dashboard-sample.json
```

Comment:
Maybe use the relative base syntax? https://dotnet.github.io/docfx/tutorial/links_and_cross_references.html#relative-path-vs-absolute-path

Reply:
Hmm... do you mean the ~ character by the relative base syntax?
Doesn't it require to have DocFX in place?

Reply:
Never mind, there doesn't seem to be a relative base support https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links.